docs/getting-started.md

```diff
@@ -197,8 +197,9 @@ sudo ip link set dev "$TAP_DEV" up
 
 # Enable ip forwarding
 sudo sh -c "echo 1 > /proc/sys/net/ipv4/ip_forward"
+sudo iptables -P FORWARD ACCEPT
 
-HOST_IFACE="eth0"
+HOST_IFACE=$(ip -j route list default |jq -r '.[0].dev')
```

Comment:
This may not work for everyone either. Maybe leave a comment what it's doing so everyone could assess if it's true for them?

Reply:
fair, done!